compositor_render/src/transformations/shader_params.rs

```diff
@@ -18,15 +18,15 @@ impl ParamsBuffer {
         let buffer = wgpu_ctx
             .device
             .create_buffer_init(&wgpu::util::BufferInitDescriptor {
-                label: Some("builtin node params buffer"),
+                label: Some("shader params buffer"),
```

Comment:
This is misleading. I know, that you refer to GPU shader, rather than to shader "transformation", but users will see in logs that suggest that they're using `shader` "transformation".